src/prometheus_alert_rules/unit_unavailable.rule

```diff
@@ -1,5 +1,5 @@
 alert: TraefikIngressUnitIsUnavailable
-expr: up{%%juju_topology%%} < 1
```

Comment:
why was this here in the first place?

Reply:
Because in ancient times we used `%%juju_topology%%` to be replaced with the actual topology... now the topology is injected and this "tag" is no longer necessary

Reply:
I thought that this kind of tag replacement is precisely what "injection" means. Maybe I need to dig up a bit better how that works

Reply:
You can take a look at [this class](https://github.com/canonical/prometheus-k8s-operator/blob/main/lib/charms/prometheus_k8s/v0/prometheus_scrape.py#L2252) and [this tool](https://github.com/canonical/cos-tool/)